docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -10,6 +10,34 @@ checked: true
 testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
 ```
 
+* [Browser List](#browser-list)
+  * [Local Browsers](#local-browsers)
+  * [Remote Browsers](#remote-browsers)
+* [File Path/Glob Pattern](#file-pathglob-pattern)
+* [Options](#options)
+  * [-h, --help](#-h---help)
+  * [-v, --version](#-v---version)
+  * [-b, --list-browsers](#-b---list-browsers)
+  * [-r \<name\>, --reporter \<name\>](#-r-name---reporter-name)
+  * [-s \<path\>, --screenshots \<path\>](#-s-path---screenshots-path)
+  * [-S, --screenshots-on-fails](#-s---screenshots-on-fails)
+  * [-q, --quarantine-mode](#-q---quarantine-mode)
+  * [-e, --skip-js-errors](#-e---skip-js-errors)
+  * [-t \<name\>, --test \<name\>](#-t-name---test-name)
+  * [-T \<pattern\>, --test-grep \<pattern\>](#-t-pattern---test-grep-pattern)
+  * [-f \<name\>, --fixture \<name\>](#-f-name---fixture-name)
+  * [-F \<pattern\>, --fixture-grep \<pattern\>](#-f-pattern---fixture-grep-pattern)
+  * [--selector-timeout \<ms\>](#--selector-timeout-ms)
+  * [--ports \<port1,port2\>](#--ports-port1port2)
+  * [--hostname \<name\>](#--hostname-name)
+  * [--qr-code](#--qr-code)
+  * [--color](#--color)
+  * [--no-color](#--no-color)
+
+> Important! Make sure to keep the browser tab that is running tests active. Do not minimize the browser window.
```

Comment:
I spent some time searching for a place to insert this note but found no better place than this, plus the `Programming Interface -> run()` topic.